### DISCUSSION TITLE
How many people do not use org-ql?

### DISCUSSION THREAD

**Discussion by toshism** [toshism/org-super-links#49] [Ideas]
[org-ql](https://github.com/alphapapa/org-ql) is really great and, in my opinion, with `helm-org-ql` by far the best interface for this type of functionality. With the recent addition of support for [custom predicates](https://github.com/alphapapa/org-ql#custom-predicates) it's just getting better and better (thanks [alphapapa](https://github.com/alphapapa)). I'm curious though, how many people who use `org-super-links` do so with `sl-search-function` set to something other than `helm-org-ql`? It would also be interesting to hear any reasons you prefer something else.

I'm tempted to drop support for anything else and move to using [org-linker](https://github.com/toshism/org-linker/) as the base for `org-super-links`. This would simplify the code and make it a bit more reliable as well. That being said if there is a significant group who have reasons for not using `org-ql` then I'll reconsider.

**Comment by piater** [comment]
I use `org-super-links` with `sl-search-function` set to `sl-get-location`, and I'd like to keep it that way.

A few years ago I gave helm a serious shot. It looked highly appealing to me, and I used it routinely for 3 or 4 months. During this time I tried very hard to like it, but in the end I removed it from my system as it kept getting in my way. I think this comes down to two reasons: First, the number keystrokes, which helm seeks to reduce, is not necessarily the best performance metric. Helm reduces keystrokes at the cost of increasing visual search among the narrowing set of options. To me, being very fast with my fingers, this turned out to be counterproductive. Secondly, this effect is probably exacerbated by the fact that over time, I subconsciously adapted my file (and other) naming conventions to fit vanilla emacs' tab completion well.

So, while I understand that helm appeals to many people, it does not work well for me.

**Reply by toshism** [reply]
that's reasonable. thanks for the feedback.

**Reply by alejandroscp** [reply]
Hola a todos, tal vez esté perdido. Seré breve, tengo un sitio [web ](url=https://comparadorglobal.com/los-mejores-ordenadores-portatiles-de-2022/) al que quiero generar backlinks de calidad, no sé si está es una herramienta para eso (white hat) y confiable. Soy neófito en Github en cuanto a interpretar los códigos, mis disculpas y gracias
